src/pyxplor/plot_time_series.py

```diff
@@ -0,0 +1,69 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+
+def plot_time_series(input_df: pd.DataFrame, 
+
```

Comment:
remove this space and align the arguments

Reply:
I didn't understand this suggestion

Reply:
this line is empty. just remove this line.

Reply:
Done